guides/common/modules/proc_registering-a-host.adoc

```diff
@@ -4,10 +4,10 @@
 You can register a host by using registration templates and set up various integration features and host tools during the registration process.
 
 .Prerequisites
-* Your user account has a role assigned that has the `create_hosts` permission.
+* Your user account has a role assigned that grants the `create_hosts` permission.
 * You must have root privileges on the host that you want to register.
 ifdef::satellite,orcharhino,katello[]
-* {ProjectServer}, any {SmartProxyServers}, and all hosts must be synchronized with the same NTP server, and have a time synchronization tool enabled and running.
+* You have configured the host for registration.
```

Comment:
Can we make it as a link to the `guides/common/modules/proc_configuring-a-host-for-registration.adoc` section